comp150-agile/essay.tex

```diff
@@ -0,0 +1,65 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Do Daily Scrums Meetings Improve Interpersonal Relations Within a Game Development Team?}
+
+% Please do not change the subtitle
+\subtitle{COMP150 - Agile Development Practice}
+
+% Please put your student number in the author field
+\author{1607804}
+
+\begin{document}
+
+\maketitle
+
+
+\section{Introduction}
+There is limited research on the topic of Daily scrum meetings and whether they improve the interpersonal relations of the team. However there is a lot of theory around team effectiveness across many work disciplines which can be extrapolated to fit the principles of Agile Software Development \cite{Agile}. 
+
+In the following essay the author will outline '‘Daily Scrum meetings' this will then be followed by the main discussion which will be based upon any relevant research and the authors own conclusions. 
+
+The author, by drawing parallels between research around team working and existing literature of '‘Daily scrum meetings', hopes to demonstrate how scrum meetings can be used to achieve improved interpersonal relations. 
+
+The Author will compare research around possible positive effects of daily scrum meetings on the team and balance this with negative outcomes. 
+

```

Comment:
Single-sentence paragraphs are to be avoided! A bare minimum of *three* sentences to form a paragraph. I suggest the use of either the [Point-Evidence-Explanation](http://www.stbonifaces.com/sites/default/files/user-31/PEE%20Paragraphing%20and%20Literacy.pdf) structure or the [State-Elaborate-Exemplfify-Illustrate](https://en.wikipedia.org/wiki/SEE-I) structure throughout your work.

Take care to link points from the previous paragraph to the next to create a sense of flow, through which your argument emerges.